### PR TITLE
JavaScript Group Capstone Project

### PR DESCRIPTION
**Pull request by Muhammad0602**
### For this capstone project we implemented the following features to our project.
- Chose the TV maze API as our base API.
- Set up the project with `webpack`, `linters`, and `jest`
- Added the `footer` and `header` sections.
- Added items on the homepage and popup window using base API.
- Added features of adding a new like and comment and also displaying them on the page by combining base and involvement APIs
- We also added counters for comments, likes, and items on the page.
- Using Jest, we wrote some unit tests for our counters.
- We added some styling including some transitions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,8 +78,10 @@ git clone github.com/Muhammad0602/Portfolio-mobile-first.git
 
 ### Usage
 
-To run the project, execute the following command:
- open the project by launching the Live Server plugging in visual studio code(or your code editor) 
+To run the project, execute the following commands:
+ - first `npm install`
+ - second `npm run build`
+ - and lastly `npm start`
```
Comment (elisha2kyakpo1) [inline README.md:85]: - [ ] Let's add the testing instructions in your readme file. I tried `jest` commands, but it did not work.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Muhammad0602 and @ghreza-crypto, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
👍 The project works as expected.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 2]
```diff
@@ -1,26 +1,56 @@
...
+    "@babel/plugin-transform-modules-commonjs": "^7.21.2",
+    "babel-eslint": "^10.1.0",
     "css-loader": "^6.7.3",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "gh-pages": "^5.0.0",
+    "hint": "^7.1.4",
     "html-webpack-plugin": "^5.5.0",
+    "jest-environment-jsdom": "^29.5.0",
```
Comment (rloterh) [inline package.json:34]: - [ ] [OPTIONAL]
Good job with the changes, however, I still had to add the lines below in the _package.json_ after installation in order for the jest to run correctly:
```javascript
"jest": {
    "testEnvironment": "jsdom"
  },
```

File: README.md [hunk 3]
```diff
@@ -76,8 +78,11 @@ git clone github.com/Muhammad0602/Portfolio-mobile-first.git
 
 ### Usage
 
-To run the project, execute the following command:
- open the project by launching the Live Server plugging in visual studio code(or your code editor) 
+To run the project, execute the following commands:
+ - first `npm install`
+ - second `npm run build`
+ - third `npm start`
+ - and lastly for test purposes `npm install --save-dev jest-environment-jsdom`
```
Comment (rloterh) [inline README.md:86]: - [ ] [OPTIONAL]
Kindly include appropriate instructions in your ReadMe on how to run test cases your app, under the _Getting Started_ section, for instance using the command:
`npm test` or `npx jest .`

**Review by rloterh — APPROVED** [review]
Hi Team, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**